package-lock.json

```diff
@@ -3,6 +3,3704 @@
   "version": "0.0.0",
   "lockfileVersion": 1,
   "requires": true,
+  "packages": {
```

Comment:
It seems that the change in package-lock.json is more than necessary.
Delete both the npm-modules folder and package-lock.json, then run npm install

Reply:
Since the value of 'lockfileVersion' is being forced to 1 by 'Bump PR', we would like to use version 14.XX of NodeJS.